spinner/spinner.go

```diff
@@ -0,0 +1,13 @@
+package spinner
+
+// This is a wrapper around spinner to support different operation systems until upstream is fixed.
+
+func New() SpinnerWrapper {
+	return new()
+}
+
+// SpinnerWrapper is used to abstract a spinner so that it can be conveniently disabled on Windows.
```

Comment:
I would say "on terminals which don't support it" instead of "on Windows". This could change.